templates/js-standby/package.json

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "js-standby-project",
+    "version": "0.0.1",
+    "type": "module",
+    "description": "This is a boilerplate of an Apify Actor.",
+    "engines": {
+        "node": ">=18.0.0"
```

Comment:
Is it necessary to have a node js version here? If so, I would put at least the v20 as a docker image.